20210325/AGENDA.md

```diff
@@ -0,0 +1,75 @@
+        - [ ] Alba propose examples on how to program without `on_stop()` and `on_abort()` Macro methods.
+        - [ ] MAX IV will provide a docker image to add to CI.
+        - [ ] Shutter integration in continuos scans: status update.
+        - [ ] MaxIV beats configuration status: update from  Alejandro's work.
+        - [ ] Users can break Sardana by messing with configuration in Tango DB, diagnostic script should be created to check that.
+        - [ ] Configure CI on buster (problems with test suite on Python 3.6): status update..
+        - [ ] Verify if Sardana axis/controller attribute are memorized by default.
+        - [ ] Hang scan due to the counter/timer controller timeout in StateOne() - How to deal with exceptions in the controllers: Review the docs?
+6. Overview of current developments / PR distribution
+    - Apply position formatting to the limits [#1530](https://github.com/sardana-org/sardana/pull/1530)
+    - Add server_init_hook [#1532](https://github.com/sardana-org/sardana/pull/1532)
+    - Respect timer/monitor passed in measurement group configuration [#1521](https://github.com/sardana-org/sardana/pull/1521)
+    - Problems for axis attributes with same name but different properties [#1436](https://github.com/sardana-org/sardana/issues/1436)
+    - Allow SWMR mode and avoid flock problems with NXscanH5_FileRecorder [#1457](https://github.com/sardana-org/sardana/issues/1457). Carlos will review it.
+    - SEP19. Update from Zbi.
+    - pre- and post-move hooks not available in mv macro [#1471](https://github.com/sardana-org/sardana/issues/1471). A propostion: add pre-mv and post-mv to mv, have sardana custom setting to  call them in a mv or not.
+      Meanwhile work on general hooks with more granularity, add hooks to macros or groups of macros.
+    - 1D/2D shape should go out from the configuration. [#1466](https://github.com/sardana-org/sardana/pull/1466/files#diff-b4dc204bf8202495936aa3777355984035597d4d9da04f35dbe9342c312782a5R666). Already merged.
+    - RoIs settings set in a file. A PR was merged.
+    - Add script to upgrade mntgrp from Sardana 2 to Sardana 3 - [#1488](https://github.com/sardana-org/sardana/pull/1488)
+    - WIP: Add hook execution logging - [#1496](https://github.com/sardana-org/sardana/pull/1496)
+    - WIP: Make Sardana storage Tango independent - [#1495](https://github.com/sardana-org/sardana/pull/1495)
+    - WIP: Add RoI pseudo counters [#1482](https://github.com/sardana-org/sardana/pull/1482)
+    - WIP: Generic data recorder [#1478](https://github.com/sardana-org/sardana/pull/1478)
+    - Sardana upload to conda-forge
+    - Shape is read only axis parameter
+    - Documentation was improved: SEP2 and SEP18
+    - All GUI related issues were marked with a label - soon a student from ALBA  - Gabriel will start working on them
+    
+7. Migration to gitlab. 
```

Comment:
After my experience with the taurus migration, I feel confident that I can submit a PR with the implementation of the CI for sardana based on gitlab-CI, including:
- running test suite
- building and deploying docs

Would that be welcome?
- building and deploying to pypi

Reply:
Hi,
thanks for your comment.

As far as I remember our decision was to stay at the moment with Github Actions and keep Gitlab as an alternative for future. But IMHO sharing your experience would be beneficial to Sardana in case of migaration. What do you think @reszelaz ?

Reply:
Thanks @cpascual for your offer. We will discuss it today.